pkg/common/providers/ocmprovider/cluster.go

```diff
@@ -628,3 +638,72 @@ func (o *OCMProvider) ExtendExpiry(clusterID string, hours uint64, minutes uint6
 
 	return nil
 }
+
+// AddProperty adds a new property to the properties field of an existing cluster
+func (o *OCMProvider) AddProperty(clusterID string, tag string, value string) error {
+	var resp *v1.ClusterUpdateResponse
+
+	// Get the current state of the cluster
+	ocmCluster, err := o.getOCMCluster(clusterID)
+
+	if err != nil {
+		return err
+	}
+
+	cluster, err := o.ocmToSPICluster(ocmCluster)
+	if err != nil {
+		return err
+	}
```

Comment:
We can probably get rid of this -- I don't think we need to do this conversion here.

Reply:
Yeah I guess this can be removed without affecting functionality.